examples/simple.py

```diff
@@ -4,102 +4,86 @@
 from flashbots import flashbot
 from flashbots.types import SignTx
 from eth_account.account import Account
-from web3 import Web3, HTTPProvider
+from web3 import Web3, HTTPProvider, exceptions
 from web3.types import TxParams, Wei
 
 import os
+import requests
+import math
 
 """
-In this example we setup a transaction for 0.1 eth with a gasprice of 1
-From here we will use Flashbots to pass a bundle with the needed content
+In this example we setup a transaction for 0.1 eth with an appropriate gasprice.
+From here we will use Flashbots to pass a bundle with the needed content.
 """
-ETH_ACCOUNT_SIGNATURE: LocalAccount = Account.from_key(
-    os.environ.get("ETH_SIGNATURE_KEY")
-)
-ETH_ACCOUNT_FROM: LocalAccount = Account.from_key(os.environ.get("ETH_PRIVATE_FROM"))
-ETH_ACCOUNT_TO: LocalAccount = Account.from_key(os.environ.get("ETH_PRIVATE_TO"))
 
-print("Connecting to RPC")
-# Setup w3 and flashbots
-w3 = Web3(HTTPProvider("http://localhost:8545"))
-w3.middleware_onion.add(construct_sign_and_send_raw_middleware(ETH_ACCOUNT_FROM))
-flashbot(w3, ETH_ACCOUNT_SIGNATURE)
+if not os.environ.get("SEND_TO") or not os.environ.get("ETH_PRIVATE_KEY"):
+    print("env variables ETH_PRIVATE_KEY and SEND_TO required")
+    exit(1)
 
-print(
-    f"From account {ETH_ACCOUNT_FROM.address}: {w3.eth.get_balance(ETH_ACCOUNT_FROM.address)}"
-)
-print(
-    f"To account {ETH_ACCOUNT_TO.address}: {w3.eth.get_balance(ETH_ACCOUNT_TO.address)}"
-)
+# signifies your identify to the flashbots network
+FLASHBOTS_SIGNATURE: LocalAccount = Account.create()
+ETH_ACCOUNT: LocalAccount = Account.from_key(os.environ.get("ETH_PRIVATE_KEY"))
+SEND_TO: str = os.environ.get("SEND_TO") # Eth address to send to
 
-# Setting up an transaction with 1 in gasPrice where we are trying to send
-print("Sending request")
-params: TxParams = {
-    "from": ETH_ACCOUNT_FROM.address,
-    "to": ETH_ACCOUNT_TO.address,
+print("connecting to RPC")
+w3 = Web3(HTTPProvider("http://localhost:8545"))
+w3.middleware_onion.add(construct_sign_and_send_raw_middleware(ETH_ACCOUNT))
+flashbot(w3, FLASHBOTS_SIGNATURE)
+
+print(f"account {ETH_ACCOUNT.address}: {w3.eth.get_balance(ETH_ACCOUNT.address)} wei")
+
+# the bribe is contained in the gas price when not using a custom smart contract.
+# it must be high enough to make all the transactions in the bundle have a 
+# competative affective average gas price 
+def get_gas_price():
+    gas_api = 'https://ethgasstation.info/json/ethgasAPI.json'
+    response = requests.get(gas_api).json()
+
+    gas_multiplier = 10
+    gas_price_gwei = math.floor(response["fastest"] / 10 * gas_multiplier)
```

Comment:
You are spending 10x the gas cost here...

Reply:
probably overkill, I'll change to 3x

Reply:
I say remove the multiplier because a $20 transaction is $60 at 3x and it's the fastest setting so the transaction should get picked up when broadcasted to the next 3 blocks.